crates/requirements-txt/src/lib.rs

```diff
@@ -386,13 +390,14 @@ impl RequirementsTxt {
                     end,
                 } => {
                     let sub_file = requirements_dir.join(filename);
```

Comment:
So the join here is fine even if `filename` is a URL?

Reply:
Yup works really well from my testing. It is basically handled like a relative path starting with the `http:` directory. The only sketchy thing I could find for now is that it doesn't survive a round trip through `.components()`:
```PathBuf::from_iter(PathBuf::from_str("http://test/abc.txt").unwrap().components())``` = `http:/test/abc.txt` (one `/` missing).